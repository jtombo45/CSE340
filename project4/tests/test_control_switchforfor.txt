a, b, c, d, e;
{
	b = 1;
	a = 2;
	SWITCH a {
	CASE 2: { c = 2*3; }
	CASE 3: { c = 4*5; }
	DEFAULT: { print b; c = 100; }
	}

	FOR(b=1;b<5;b=b+1;) {
		d = 2*b;
		FOR(e=0; e<6; e=e+2;) {
			print e;
		}
		print d;
	}
	print c;
}