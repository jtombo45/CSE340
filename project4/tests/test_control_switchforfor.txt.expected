0
2
4
2
0
2
4
4
0
2
4
6
0
2
4
8
6